src/tools/storage/s3/amazon-server-side/s3UploaderFunction/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "none",
+  "private": true,
+  "version": "1.0.0",
+  "description": "",
+  "main": "app.js",
+  "scripts": {
+    "start": "node testHarness.js",
+    "pack": "zip -r ../.infrastructure/lambda.zip ./",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "license": "UNLICENSED",
```

Comment:
I see an "UNLICENSED" license in this packages.json file, which would make it no longer open-source software. Can someone elaborate on it?

Reply:
Sorry, forget this field. 
I guess it will be GPL v3.
Will fill licence, package name description etc.